.rhcicd/clowdapp-connector-drawer.yaml

```diff
@@ -128,6 +132,12 @@ parameters:
   value: 250Mi
 - name: MIN_REPLICAS
   value: "3"
+- name: NOTIFICATIONS_CONNECTOR_HTTP_CONNECT_TIMEOUT_MS
+  description: Maximum time in milliseconds allowed to establish an HTTP connection
+  value: "2500"
+- name: NOTIFICATIONS_CONNECTOR_HTTP_SOCKET_TIMEOUT_MS
+  description: Maximum time in milliseconds allowed to wait for HTTP data
+  value: "180000"
```

Comment:
This is the default value from Camel: https://camel.apache.org/components/4.0.x/http-component.html#_component_option_soTimeout